File: src/main/org/nlogo/window/ClientAppInterface.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.window

import org.nlogo.api.ParserServices

trait ClientAppInterface {
  def startup(editorFactory: EditorFactory, userid: String, hostip: String, port: Int,
    isLocal: Boolean, isRobo: Boolean, waitTime: Long, workspace: ParserServices): Unit
}

File: src/main/org/nlogo/window/ClientAppInterface.java
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.window;

import org.nlogo.api.ParserServices;

public interface ClientAppInterface {
  void startup(final EditorFactory editorFactory, final String userid, final String hostip,
               final int port, final boolean isLocal, final boolean isRobo, final long waitTime,
               final ParserServices workspace);
}
